### PR TITLE
Add dotnet server example

### PR DESCRIPTION
**Pull request by cjavilla-stripe**
r? @dawn-stripe 

Making this a PR into the go one because I'm re-using the same test.

Also found some issues with node but don't have bandwidth to fix them all right now :(

### CODE REVIEW

**Comment by dawn-stripe** [comment]
Thanks for tackling this!  I have a question about the following files, I don't see them in the fixed price sample, so I'm not sure if they should be here (they seem in the 'user dev config' realm), or if we should be adding them to that sample/ensuring we're including them?

.vs/dotnet/xs/UserPrefs.xml
obj/*.*

**Comment by cjavilla-stripe** [comment]
hmm. yeah I think I need to add those userprefs to gitignore. I think those were added by vs for mac :/

File: per-seat-subscriptions/server/dotnet/Models/CreateSubscriptionRequest.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    public string PaymentMethod { get; set; }
+
+    [JsonRequired()]
+    [JsonProperty("customerId")]
+    public string Customer { get; set; }
+
+    [JsonRequired()]
+    [JsonProperty("priceId")]
+    public string Price { get; set; }
+}
```
Comment (jrondeau-stripe) [inline per-seat-subscriptions/server/dotnet/Models/CreateSubscriptionRequest.cs:16]: Do we need a quantity here too?

File: per-seat-subscriptions/server/dotnet/Controllers/BillingController.cs [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+
+            // Create subscription
+            var subscriptionOptions = new SubscriptionCreateOptions
+            {
+                Customer = req.Customer,
+                Items = new List<SubscriptionItemOptions>
+                {
+                    new SubscriptionItemOptions
+                    {
+                        Price = Environment.GetEnvironmentVariable(req.Price),
```
[the remark this answers is not in the document]
Comment (cjavilla-stripe) [reply per-seat-subscriptions/server/dotnet/Controllers/BillingController.cs:116]: yep. great catch! I think the new tests we added should catch this. I'll run again and get this fixed.

File: per-seat-subscriptions/server/dotnet/Models/CreateSubscriptionRequest.cs (hunk 1 above)
Comment (cjavilla-stripe) [reply per-seat-subscriptions/server/dotnet/Models/CreateSubscriptionRequest.cs:16]: yep

Comment (cjavilla-stripe) [reply per-seat-subscriptions/server/dotnet/Models/CreateSubscriptionRequest.cs:20]: nice catch

**Comment by cjavilla-stripe** [comment]
Merging to get docs out, but we can iterate if this still needs changes. Tests are passing on dotnet now.

**Pull request merged**